README.md

```diff
@@ -1,15 +1,52 @@
 ### PyPi Mirror Manager
 
-![](http://ofydnun2c.bkt.clouddn.com/ezgif.com-gif-maker.gif)
+[![command line demo](https://asciinema.org/a/3e67r6npdb4i7syuqdmxkctgm.png)](https://asciinema.org/a/3e67r6npdb4i7syuqdmxkctgm)
 
-#### Install
+#### Installation
 
-```
-pip install pmm
-```
+    pip install pmm
+
+#### Usage
+
+    usage: pmm [-h] [-m]
+
+    Select PyPI index server used by pip.
+
+    optional arguments:
+      -h, --help     show this help message and exit
+      -m, --mirrors  download list of PyPI mirrors and add them to selection
+
+#### Configuration
+
+You can add package indexes to your `pip.conf` file. Example:
+
+    [global]
+    use-wheel = True
+    index-url = https://pypi.python.org/simple
+    index-servers =
+        pypi
+        pypi-test
+        my-devpi
+
+    [pypi]
+    index = pypi.python.org
+
+    [pypi-test]
+    index = testpypi.python.org/pypi
+
+    [my-devpi]
+    index = devpi.example.com/main/dev
+    location = Example Inc., California US
```

Comment:
yeah I think we can rename it to `info`, which is more general, and the locations returned by pypi-mirros is just a kind of info, the user can specify something else.